components/SplitDisplay.vue

```diff
@@ -117,10 +119,10 @@ export default class SplitDisplay extends Vue {
 
   get bestTimeDisplay() {
     let out = '';
-    if (this.split.BestSegmentTime.GameTime)
-      out += `${this.formatTime(this.split.BestSegmentTime.GameTime)} (game time) ; `;
-    if (this.split.BestSegmentTime.RealTime)
+    if (store.state.useRealTime)
```

Comment:
I think we can just use `selectTime` and remove the parenthesis (`(game/real time)`), instead of having these 2 ugly `if`s